components/brave_federated_learning/brave_operational_profiling_features.cc

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_federated_learning/brave_operational_profiling_features.h"
+
+#include "base/metrics/field_trial_params.h"
+
+namespace operational_profiling {
+namespace features {
+
+namespace {
+const char kFeatureName[] = "FederatedLearningOperationalProfiling";
+
+const char kFieldTrialParameterCollectionSlotSizeInMinutes[] =
+    "collection_slot_size_in_minutes";
+const int kDefaultCollectionSlotSizeInMinutes = 30;
+
+const char kFieldTrialParameterSimulateLocalTrainingStepDurationInMinutes[] =
+    "simulate_local_training_step_duration_in_minutes";
+const int kDefaultSimulateLocalTrainingStepDurationInMinutes = 5;
+
+const char kFieldTrialParameterEphemeralIDLifetimeInDays[] =
+    "ephemeral_id_lifetime_in_days";
+const int kDefaultEphemeralIDLifetimeInDays = 30;
```

Comment:
Wouldn't it be better to have a default of 1 day (daily) for ephemeral ID lifetime? We are controlling these via Griffin as well, right?

Reply:
Changed it 👍